src/keycloak/chart/templates/istio-admin.yaml

```diff
@@ -36,18 +35,6 @@ spec:
             notNamespaces:
             - istio-admin-gateway
             - monitoring
-    - to:
-        - operation:
-            ports:
-              - "8080"
-            paths:
-              # Never allow anonymous client registration except from the pepr-system namespace
-              # This is another fallback protection, as the KC policy already blocks it
-              - "/realms/{{ .Values.realm }}/clients-registrations/*"
-      from:
-        - source:
-            notNamespaces:
-              - "pepr-system"
```

Comment:
Do we need an upgrade/migration on the realm/identity-config side to make sure this endpoint is no longer accessible/turned on?

Reply:
No, the migration should happen within UDS Core v0.39.0, which pulled the v0.11.0. Here's the upgrade docs: https://uds.defenseunicorns.com/reference/uds-core/idam/upgrading-versions/#v0110

Reply:
I knew client credentials got _enabled_ in that migration but I missed that the dynamic endpoint/access was also _disabled/removed_. I did validate this locally, when I try to hit the dynamic endpoint I end up getting this log/error: `{"error":"Client registration provider not found"}`, although I'm still a bit unsure where/how that changed (this is mostly curiosity, now that I have validated it's not accessible anymore).

Reply:
This is very weird... 

It basically means that all `ClientRegistrationProvider` implementations have been dropped from the Realm. There should be at least 2 of them - the default and the OIDC one. 

This error definitely should not happen and it leads to a question if your UDS Realm has been properly initiated?

Reply:
I'm resolving this as we clarified in Slack that `{"error":"Client registration provider not found"}` error comes from the wrong URL used. The URL needs to end with "default" as we're testing the "default" Dynamic Client Registration API.